filters/builtin/timeout.go

```diff
@@ -31,15 +65,34 @@ func (*timeout) CreateFilter(args []interface{}) (filters.Filter, error) {
 		tf.timeout = d
 	case time.Duration:
 		tf.timeout = v
+	case int:
+		tf.timeout = time.Duration(v) * time.Millisecond
```

Comment:
Lets not support untyped integer and require either duration string (eskip) or time.Duration (for lib users).
See reasoning here https://github.com/zalando/skipper/pull/1814#issuecomment-895053332

Reply:
Then untyped will be `ns`, which I think is bad. There was no discussion or agreement on your cited comment...
latency() is also having int -> value in ms. I don't see the problem why we should change this.

Reply:
> Then untyped will be ns, which I think is bad

Why do you think so? I propose we do not support integer at all, only duration strings like "1s" or time.Duration value for lib users to eliminate any room for millisecond vs second mistake

Reply:
fixed